typescript-timers/interval.js

```diff
@@ -0,0 +1,14 @@
+let value = document.querySelector('.countdown-display');
+
+function timer() {
+  let currentValue = value.textContent;
+  currentValue--;
+  value.innerHTML = currentValue;
```

Comment:
@claylevy53 As a habit, please use `textContent` instead of `innerHTML` to avoid security issues.